salt/metalk8s/addons/prometheus-operator/deployed/chart.sls

```diff
@@ -25089,13 +25089,12 @@ data:
   datasource.yaml: |-
     apiVersion: 1
     datasources:
-    - name: Prometheus
+    - access: proxy
+      isDefault: true
+      name: Prometheus
       type: prometheus
+      uid: metalk8s-prometheus
       url: http://thanos-query-http:10902/
-      access: proxy
-      isDefault: true
-      jsonData:
-        timeInterval: 30s
```

Comment:
This was lost :confused:

Reply:
Hmm I thought it was the default from grafana but it's not apparently, I will add it back :+1: